Session.h

```diff
@@ -39,7 +40,7 @@ class Session {
  public:
     std::string uuid;
     carta::FileSettings fsettings;
-    tbb::concurrent_queue<std::pair<CARTA::SetImageChannels,uint32_t>> aniq;
+    tbb::concurrent_queue<std::pair<CARTA::SetImageChannels,uint32_t>> setchanq;
```

Comment:
I presume once we have animation messages working properly, we can limit this queue to a size of one, since _only_ the latest SET_CHANNELS message will be valid

Reply:
Yes, this could just become a pointer.

Reply:
Yes could just be a pointer once it is not used for animation any more.